common/models/User.php

```diff
@@ -89,6 +89,7 @@ public function rules()
             [['username', 'email'], 'unique'],
             ['status', 'default', 'value' => self::STATUS_ACTIVE],
             ['status', 'in', 'range' => [self::STATUS_ACTIVE, self::STATUS_DELETED]],
+            [['username','email','password_hash'],'filter','filter'=>'\yii\helpers\Html::encode']
```

Comment:
Hi,
Its better to have it for email too since we show that field in the
frontend,and is user input, so I think we should keep that.

Regarding the password , I am not sure if we ever show that (we
shouldn't!). If that's the case I will remove it.

Let me know if you want me to remove the filter for both or just for
password, I can send an updated pull request in about 30 minutes.

Romina
El sep 3, 2015 4:21 PM, "Eugene Terentev" <notifications@github.com>
escribió:

> In common/models/User.php
> <https://github.com/trntv/yii2-starter-kit/pull/280#discussion_r38685577>:
>
> > @@ -89,6 +89,7 @@ public function rules()
> >              [['username', 'email'], 'unique'],
> >              ['status', 'default', 'value' => self::STATUS_ACTIVE],
> >              ['status', 'in', 'range' => [self::STATUS_ACTIVE, self::STATUS_DELETED]],
> > +            [['username','email','password_hash'],'filter','filter'=>'\yii\helpers\Html::encode']
>
> email and hash too? why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trntv/yii2-starter-kit/pull/280/files#r38685577>.
>